common/classify.py

```diff
@@ -90,12 +92,16 @@ def test_and_report(self):
          - test.mat has data['email_index']
         Results is [path, index, probability]
         """
-        self.clean_all() 
+        self.clean_all()
         if not os.path.exists(self.results_dir):
-            os.makedirs(self.results_dir) 
+            os.makedirs(self.results_dir)
+
+        lowVolumeTop10 = PriorityQueue()
```

Comment:
nit: I think it would be clearer if we stuck with snake case for variable names.